src/plugins/intel_npu/src/backend/src/zero_infer_request.cpp

```diff
@@ -131,7 +131,7 @@ std::optional<size_t> getBatchSize(
     }
 
     if (batch_size.size() != 1) {
-        logger.info("Batching works only when we have the same batch size for all tensors!");
+        logger.warning("Batching works only when we have the same batch size for all tensors!");
```

Comment:
Not sure that we want to change these batch-related loggers to warning.

Reply:
Considering that the default loglevel is set to either error or warning, it is better to provide a relevant info if the Batch functionality is not utilized correctly. 
From the[ OV's perspective on loglevel,](https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/properties.hpp#L607) using `log.info` is more suitable here, as this part does not lead to any harmful impacts that would halt the program's execution. The change to `log.info` has been made in new commit.